api/catalog/api/serializers/audio_serializers.py

```diff
@@ -135,6 +135,17 @@ def get_peaks(obj) -> list[int]:
             obj = Audio.objects.get(identifier=obj.identifier)
         return obj.get_waveform()
 
+    def to_representation(self, instance):
+        # Get the original representation
+        output = super().to_representation(instance)
+
+        if isinstance(instance, Hit):
+            audio = Audio.objects.get(identifier=instance.identifier)
```

Comment:
Given that we know this isn't a permanent solution, do you mind adding a `# TODO: Remove when <condition>` comment here so we can ensure this is removed when we make the ES changes?